### DISCUSSION TITLE
Documentation: Main Page

### DISCUSSION THREAD

**Discussion by DBartlettHP** [humanpred/pknca#322] [General]
In the sentence:
"When automation would leave ambiguity or make a choice that the analyst may have an alternate preference for, it is either not used or is possible to override." 

Will the automation also give useful error messages depending on the situation? (Perhaps a note on having "fruitful" error message reporting could be intriguing here)

---
In the sentence:
"These will be especially noticeable around the inclusion of IV NCA parameters and additional specifications of the dosing including dose amount and route."

I was reading this wrong at first. Perhaps it can be rewritten? ("...IV NCA parameters and additional specifications of the drug, which  include dose amount and route of administration")

---
In the sentence:
"More help is available in the function help files, and be sure to look at the PKNCA.options function for many choices to make PKNCA conform to your company’s business rules for calculations and summarization."

Can be rewritten to:
"More help is available in the function help files. Be sure to look at the PKNCA.options function for choices on making PKNCA conform to your company’s business rules on calculation and summarization."

**Comment by billdenney** [comment]
For the second and third, yes, please make those updates in a PR.

For the first, yes, the automated choices should always give useful error messages. If they don't, then the error messages should be improved. (I'm not sure if I answered everything you were asking about that one.)
